### DISCUSSION TITLE
Easy way to edit sessions

### DISCUSSION THREAD

**Discussion by Boloflex** [sienori/Tab-Session-Manager#1413] [Q&A]
I have a lot of saved sessions many of which have a lot of tabs and I am looking for an easy way to edit them if possible. I'd like to be able to rearrange, delete and move things to other sessions. Trying to edit them thru the browser extension seems to be incredibly tedious. Is there any easy or simple way to access the sessions file or export them to a file and then edit them? It would be really nice if there was some kind of GUI editor but perhaps I'm asking too much.

**Comment by NamitaBi** [comment]
@Boloflex Just created a new feature request for something same in line with your question: https://github.com/sienori/Tab-Session-Manager/discussions/1541 .
